### PR TITLE
feat: add ability to specify extra files for releasers to consider

### PR DESCRIPTION
**Pull request by chingor13**
Currently, only Java utilizes this option, but this could be used by
any release strategy.

Towards #305 
Towards #597

### CODE REVIEW

**Comment by chingor13** [comment]
cc @igorbernstein2 This would be a more generalized solution for being able to make replacements is one-off files (without hard-coding a list)

File: src/releasers/java-yoshi.ts [hunk 1]
```diff
@@ -299,6 +299,18 @@ export class JavaYoshi extends ReleasePR {
         })
       );
     });
+
+    this.extraFiles.forEach(path => {
```
Comment (bcoe) [inline src/releasers/java-yoshi.ts:303]: my slight concern here is behavior that's one-language specific, perhaps add a TODO to flesh out this behavior. I'm imagining you'd eventually pass a file plus the updater to apply?

Comment (chingor13) [reply src/releasers/java-yoshi.ts:303]: I think that will be too much config. The general use case for extra files would be one-offs that update the version strings in a file. Each language will want a generic way to do this - we don't want a bunch of one-off updaters for this.

**Review by bcoe — APPROVED** [review]

**Review by bcoe — APPROVED** [review]
👍 with the note that I'd like to minimize language specific features if we can make them more generic.

**Review by viv718918 — COMMENTED** [review]
ok

**Pull request merged**